interp/type.go

```diff
@@ -885,7 +888,13 @@ func isComplete(t *itype, visited map[string]bool) bool {
 		visited[name] = true
 	}
 	switch t.cat {
-	case aliasT, arrayT, chanT, chanRecvT, chanSendT, ptrT:
+	case aliasT:
```

Comment:
Sorry, because github is stupid, I have to comment here.

I believe L884-L886 can be simplified as:

```
if visited[name] {
   return true
}
```